benchmarks/ModelingToolkit/ThermalFluid.jmd

```diff
@@ -309,7 +309,7 @@ let ax = Axis(f[1, 1]; title="Structural Simplify Time")
     ts = @view(ss_times[:, j])
     lines!(N_states, ts; label)
   end
-  axislegend(ax)
+  axislegend(ax, position = :lt)
```

Comment:
what does this do?

Reply:
Covers the left half of the graph with plot labels.

Reply:
To give a more serious response, the intention was to place it in the top left (`l` for `left` and `t` for `top`), as the top left should be empty for graphs with increasing values. That is, the idea was that it should be able to fit there without covering anything up.

Idea != reality.